q2_demux/_summarize/assets/quality-plot.html

```diff
@@ -63,8 +63,23 @@ <h5 class="text-center">Forward Reads</h5>
   <div id="reverseContainer" class="col-xs-12 col-md-6">
     <h5 class="text-center">Reverse Reads</h5>
   </div>
+  <div class="row">
```

Comment:
Can you move this block ([L66](https://github.com/qiime2/q2-demux/pull/77/files#diff-fba5da0182f7db93468361264d2b2308R66) - [L75](https://github.com/qiime2/q2-demux/pull/77/files#diff-fba5da0182f7db93468361264d2b2308R75)) up into the `reverseContainer` `div` above? This will ensure that the download links are laid out in the parent container correctly, otherwise they get smashed into the corner on wide displays.

![screen shot 2018-01-23 at 9 16 00 am](https://user-images.githubusercontent.com/274668/35286754-1317b59a-001e-11e8-931e-6de28e2bf0e4.png)